CHANGES.md

```diff
@@ -1,3 +1,112 @@
+
+This release includes a fix for a bug in 0.6.0 (also fixed in 0.6.1):
+
+* [#1062](https://github.com/typelevel/cats/pull/1062): `Order` instances for tuples are now lexicographic (instead of only comparing first elements)
+
+And other bug fixes:
+
+* [#1096](https://github.com/typelevel/cats/pull/1096): `inj` and `prj` on `Inject` now work consistently with respect to `null`
+
+And some additions:
+
+* [#1225](https://github.com/typelevel/cats/pull/1225): `FunctorFilter` and `TraverseFilter`
+* [#1121](https://github.com/typelevel/cats/pull/1121): `valueOr` and `merge` for `Validated`
+* [#1188](https://github.com/typelevel/cats/pull/1188): `toValidatedNel` for `XorT`
+* [#1127](https://github.com/typelevel/cats/pull/1127): `toTry` for `Xor`
+* [#1130](https://github.com/typelevel/cats/pull/1130): `isEmpty` syntax method for `Monoid`
+* [#1167](https://github.com/typelevel/cats/pull/1167): `minimum`, `maximum`, and related helper methods for `Foldable` and `Reducible`
+* [#1041](https://github.com/typelevel/cats/pull/1041): `FlatMapRec` and `MonadRec`
+* [#1134](https://github.com/typelevel/cats/pull/1134): `cats.syntax.list` for à la carte list syntax imports
+* [#1191](https://github.com/typelevel/cats/pull/1191): `cats.syntax.monoid` for à la carte `Monoid` syntax imports
+* [#558](https://github.com/typelevel/cats/pull/588) and [#1063](https://github.com/typelevel/cats/pull/1063): `IdT`, the identity monad transformer
+* [#1021](https://github.com/typelevel/cats/pull/1021) and [#1221](https://github.com/typelevel/cats/pull/1221): `Nested` (represents nested composition of type constructors)
+* [#1172](https://github.com/typelevel/cats/pull/1172): `toNested` for `OptionT` and `XorT`
+* [#1102](https://github.com/typelevel/cats/pull/1102) and [#1170](https://github.com/typelevel/cats/pull/1170): `Comparison` (represents the result of an `Order` comparison)
+* [#1090](https://github.com/typelevel/cats/pull/1090): `Kleisli.lift`
+* [#1169](https://github.com/typelevel/cats/pull/1169): `lift`, `inspect`, and related methods for `StateT`
+* [#1114](https://github.com/typelevel/cats/pull/1114): `size` for `Foldable`
+* [#1193](https://github.com/typelevel/cats/pull/1193): `reduceLeftM` for `Reducible`
+* [#1097](https://github.com/typelevel/cats/pull/1097): Functor variance helpers (`widen` for `Functor` and `narrow` for `Contravariant`)
+* [#1207](https://github.com/typelevel/cats/pull/1207): `tell` for `Writer` and `WriterT`, `value` for `Writer`
+* [#1155](https://github.com/typelevel/cats/pull/1155): Convenience methods for constructing `XorT` values
+* [#1085](https://github.com/typelevel/cats/pull/1085): `runTailRec` and `foldLeftM` for `Free`
+
+And some name changes:
+
+* [#1140](https://github.com/typelevel/cats/pull/1140): `cats.std` is now `cats.instances`
+* [#1201](https://github.com/typelevel/cats/pull/1201): Many `*Unsafe` methods are now `unsafe*`
```

Comment:
Ah, thanks @fthomas. There's a possibility there's also stuff still missing—I'll do a closer pass this afternoon.